### PR TITLE
Foreign contacts section

### PR DESCRIPTION
**Pull request by bmallred**
Issues: #668, #861

### CODE REVIEW

File: src/components/Section/Foreign/Contacts/ForeignNational.jsx [hunk 1]
```diff
@@ -0,0 +1,574 @@
...
+                      />
+          </Field>
+        </Show>
+
+        <Field title={i18n.t('foreign.contacts.heading.frequency')}
+               help="foreign.contacts.help.frequency">
+          <RadioGroup className="frequency"
+                      selectedValue={this.state.Frequency}>
+            <Radio name="frequency-daily"
+                   label={i18n.m('foreign.contacts.label.daily')}
```
Comment (floresj) [inline src/components/Section/Foreign/Contacts/ForeignNational.jsx]: Can't remember if we do onValidate on these guys.

File: src/validators/foreigncontacts.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    if (this.hasForeignContacts === 'No') {
+      return true
+    }
+
+    if (this.hasForeignContacts === 'Yes') {
+      if (!this.list || this.list.length === 0) {
+        return false
+      }
+
+      return this.list.every(item => new ForeignNationalValidator(item.Item, null).isValid())
```
Comment (floresj) [inline src/validators/foreigncontacts.js]: Sweet. Totally going to use `every()`

File: src/validators/helpers.js [hunk 3]
```diff
@@ -97,6 +97,8 @@ export const validNotApplicable = (notApplicable, logic) => {
   // If the `NotApplicable` object is present and is applicable then apply the logic
   if (notApplicable && notApplicable.applicable) {
     return logic()
+  } else if (notLogic) {
+    return notLogic()
```
Comment (floresj) [inline src/validators/helpers.js]: (!hate) it.

Comment (bmallred) [inline src/validators/helpers.js]: Ha! Had to read that a few times. Yeah, needed a way to test the opposite in some cases.

File: src/components/Section/Foreign/Contacts/ForeignNational.jsx (hunk 1 above)
Comment (bmallred) [inline src/components/Section/Foreign/Contacts/ForeignNational.jsx]: I'll double check and at it in if we do!

**Pull request merged**